services/weblate/base-weblate-service.js

```diff
@@ -0,0 +1,21 @@
+import Joi from 'joi'
```

Comment:
Let's rename this file. The presence of `service` in the file name can be easily mistaken for an actual service class when browsing contents, so we typically follow a `{upstream service name}-base.js` file naming pattern for base class definitions (e.g. `weblate-base.js`)